app/Resources/views/default/catalogue.html.twig

```diff
@@ -0,0 +1,29 @@
+{% extends 'base.html.twig' %}
+
+{% block body %}
+    <div class="row">
+        <div class="col-md-12">
+            <div class="article">
+                <h1>Catalog of plants</h1><br>
+                    <div class="table-responsive">
+                        <table class="table table-hover">
+                            <tr>
+                                <th>Lp.</th>
+                                <th>Name</th>
+                                <th>Latin Name</th>
+                                <th></th>
+                            </tr>
+                            {% for plant in plants %}
+                                <tr>
+                                    <td>{{ plant.id }}</td>
+                                    <td>{{ plant.name }}</td>
+                                    <td>{{ plant.latinName }}</td>
+                                    <td><a href="/catalogue/{{ plant.id }}">show</a></td>
```

Comment:
do not make links this way, it can break up in the future
instead of `/catalogue/{{ plant.id }}` you should try `{{ path('route_name', {'id': plant.id}) }}`